packages/flutter/lib/src/services/system_chrome.dart

```diff
@@ -490,6 +490,15 @@ abstract final class SystemChrome {
     );
   }
 
+  static void Function(SystemUiOverlayStyle? style)? _systemUiOverlayStyleCallback;
```

Comment:
per style guide: This needs to use a typedef.

Reply:
sure. I'll do as soon as possible

Reply:
done!